tests/tests_app_examples/test_installation_commands_app.py

```diff
@@ -0,0 +1,22 @@
+import os
+from datetime import time
+
+import pytest
+from tests_app import _PROJECT_ROOT
+
+from lightning_app.testing.testing import run_app_in_cloud
+
+
+@pytest.mark.cloud
+def test_installation_commands_app_example_cloud() -> None:
```

Comment:
You would need to add this test within the CI, but it might be possible to merge it with the custom_dependencies one.